tools/bootstrap_web.sh

```diff
@@ -30,10 +30,10 @@ if [[ -x $node_dist/bin/node && `$node_dist/bin/node -v` == "$node_ver" ]]; then
   echo "skipping nodejs download. remove $node_dist to force redownload."
 else
   echo "Downloading nodejs"
-  rm -rf $node_dist
+  rm -rf "${node_dist}"
```

Comment:
The quotes are an improvement, however the added brackets seem unnecessary here and don't seem typical of bash style